src/zepto.js

```diff
@@ -678,6 +678,8 @@ var Zepto = (function() {
         $this.css(props)
       })
       if (!this.length) return null
+      if (!Zepto.contains(document.documentElement, this[0]))
```

Comment:
I was thinking simpler: http://jsperf.com/try-catch-3/2

We're just interested in the overhead of `try` vs `contains` on top of `getBoundingClientRect()`.

However my perf results are weird: sometimes the combination of `closest + getBoundingClientRect` is faster than just plain `getBoundingClientRect`. But generally it seems that the overhead is almost negligible compared to the computation required for `getBoundingClientRect`, and that `try` doesn't yield significant benefits over `contains`. We can stick with `contains` then and avoid generating methods at runtime for now.